dev-python/frozendict/frozendict-2.3.2.ebuild

```diff
@@ -20,6 +20,14 @@ S="${WORKDIR}/python-${P}"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	test? (
+		dev-python/pytest[${PYTHON_USEDEP}]
```

Comment:
Just use distutils_enable_tests pytest instead.

Reply:
`distutils_enable_tests` did not work for me because of C extension, see commit 89e4ee59e046 - `dev-python/frozendict: fix test phase`, where I followed https://projects.gentoo.org/python/guide/test.html#importerrors-for-c-extensions in order to fix it.

Reply:
Ah, I see, I can use `distutils_enable_tests pytests` with own `python_test` function. I'll change it.

Reply:
Done